assets/TWO_VIEW_ESTIMATOR.md

```diff
@@ -8,4 +8,73 @@
 - **Two View Estimator**
 - [Multiview Optimizer](assets/MULTIVIEW_OPTIMIZER.md)
 
-## What is a Two-View Estimator?
\ No newline at end of file
+## What is a Two-View Estimator?
+
+
+Two-View Estimator (TVE) takes information about two images and tries to determine their relative pose (how one camera is positioned and oriented with respect to the other one). It also generates correspondences between keypoint in the images.
```

Comment:
The correspondences are generated in the correspondence_generator, which is outside the two-view estimator. the config mirrors this structure: https://github.com/borglab/gtsfm/blob/master/gtsfm/configs/unified.yaml